apollo-normalized-cache/src/commonMain/kotlin/com/apollographql/apollo3/cache/normalized/ClientCacheExtensions.kt

```diff
@@ -310,14 +317,61 @@ fun <T> MutableExecutionOptions<T>.storePartialResponses(storePartialResponses:
 )
 
 /**
- * @param storeReceiveDate Whether to store the receive date.
+ * @param storeReceiveDate Whether to store the receive date in the cache.
  *
  * Default: false
  */
+@ApolloExperimental
 fun <T> MutableExecutionOptions<T>.storeReceiveDate(storeReceiveDate: Boolean) = addExecutionContext(
     StoreReceiveDateContext(storeReceiveDate)
 )
 
+/**
+ * @param storeExpirationDate Whether to store the expiration date in the cache.
+ *
+ * The expiration date is computed from the response HTTP headers
+ *
+ * Default: false
+ */
+@ApolloExperimental
+fun ApolloClient.Builder.storeExpirationDate(storeExpirationDate: Boolean): ApolloClient.Builder = apply {
+  addExecutionContext(StoreExpirationDateContext(storeExpirationDate))
```

Comment:
I think this is unused but maybe I'm missing something?

Reply:
You are 100% right. The initial idea was to make this configurable but I lost track at some point. Looking into this.